CHANGES.md

```diff
@@ -1,4 +1,4 @@
-# RELEASE DATE HERE
+# 2018/01/18
```

Comment:
This should be keyed to releases I think, not dates. If someone depending on google-cloud-java is choosing a version to depend on, they have no idea what date they were released on.

Reply:
good point; done